### ISSUE TITLE
tflite android gpu delegate init error, what does it mean? MEAN: Index for axis out of range

### ISSUE THREAD

**Issue by csjiyw** [tensorflow/tensorflow#45142]
error log:
java.lang.IllegalArgumentException: Internal error: Failed to apply delegate: TfLiteGpuDelegate Init: MEAN: Index for axis out of range
    TfLiteGpuDelegate Prepare: delegate is not initialized
    Node number 58 (TfLiteGpuDelegateV2) failed to prepare.

when i run my own model producing by tensorflow 2.3, all keras layers.

device: xiaomi 8 se
dependence: 
 implementation('org.tensorflow:tensorflow-lite:0.0.0-nightly')
 implementation('org.tensorflow:tensorflow-lite-gpu:0.0.0-nightly')
 implementation('org.tensorflow:tensorflow-lite-support:0.0.0-nightly')

**Comment by amahendrakar** [comment]
@csjiyw,
Could you please provide the exact sequence of commands and complete code that you executed before running into the problem, so that we can look into this? Thanks!

**Comment by impjdi** [comment]
we identified and fixed the bug.  please sync past commit 8206491e82f6614fa2daa314e1ec8b8ca0ed946d
